static/app/views/releases/list/index.tsx

```diff
@@ -244,6 +248,61 @@ class ReleasesList extends AsyncView<Props, State> {
     );
   }
 
+  renderAlertBanner() {
+    const {selection, organization} = this.props;
+
+    const selectedProjectId =
+      selection.projects && selection.projects.length === 1 && selection.projects[0];
+    const selectedProject = organization.projects.find(
```

Comment:
```suggestion
    const selectedProject = organization.projects?.find(
```

Reply:
👍

Reply:
This does not seem to be yet implemented.